rust/threshold-schnorr/README.md

```diff
@@ -142,148 +136,58 @@ Ed25519 public key.
 }
 ```
 
-
```

Comment:
I have the impression that the new text for this Rust threshold-schnorr example README was copy/pasted from the respective Motoko variant and adapted, but that is actually not correct as the two examples function somewhat different.

For example, the Motoko example mentions a way to test the canister with Javascript, while the Rust example does all the testing directly in Rust.

It seems all changes in this file from here on should be discarded. (Granted, there a few issues in the text, but these seem minor; for example `lib.rs` should be `wasm_only.rs`.)